src/Parser/GuardianDruid/Modules/Features/FrenziedRegenGoEProcs.js

```diff
@@ -21,6 +21,9 @@ class FrenziedRegenGoEProcs extends Module {
   statistic() {
     const nonGoEFRegen = this.guardianOfElune.nonGoEFRegen;
     const GoEFRegen = this.guardianOfElune.GoEFRegen;
+    if (nonGoEFRegen + GoEFRegen === 0) {
```

Comment:
Please use parenthesis so it's clear it's `(nonGoEFRegen + GoEFRegen) === 0`, not `nonGoEFRegen + (GoEFRegen === 0)`